.github/workflows/pr.yml

```
@@ -0,0 +1,454 @@
+            feedback += '1. Request review from maintainers\n';
+            feedback += '2. Address any review feedback\n';
+            feedback += '3. Merge when approved\n';
+          }
+          
+          // Comment on PR
+          await github.rest.issues.createComment({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            issue_number: context.issue.number,
+            body: feedback
+          });
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)